src/screens/SearchScreen.js

```diff
@@ -37,42 +40,43 @@ const SearchScreen = () => {
           term={term}
           onTermChange={(newTerm) => setTerm(newTerm)}
           onTermSubmit={() => searchApi(term, city)}
+          onEditStart={() => setTerm('')}
         />
         <CityButton 
-          toOpenModal={() => setModalVisible(true)}
+          onPress={() => setModalVisible(true)}
         />
```

Comment:
Change the names of simple callback functions to 'onPress', to make things more simple.